keyboards/sofle_choc/keymaps/default/config.h

```diff
@@ -0,0 +1,95 @@
+/* Copyright 2022 Brian Low
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#pragma once
+
+// Plug USB into the left half of the keyboard
+// see https://docs.qmk.fm/#/feature_split_keyboard?id=setting-handedness
+#define MASTER_LEFT
+
+// Enabling this option changes the startup behavior to listen for an
+// active USB communication to delegate which part is master and which
+// is slave. With this option enabled and theres’s USB communication,
+// then that half assumes it is the master, otherwise it assumes it
+// is the slave.
+//
+// I've found this helps with some ProMicros where the slave does not boot
+#define SPLIT_USB_DETECT
+
```

Comment:
Can we move ENCODER_RESOLUTION here to prevent redeclare error?
```suggestion
#define ENCODER_RESOLUTION 2

```